source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -29,9 +29,11 @@
 #include <aspect/compat.h>
 
 
+
 /**
- * This geometry model implements an (3d) ellipsoidal_chunk geometry which can be non-coordinate parallel.
- * @author This plugin is a joined effort of Menno Fraters, D Sarah Stamps and Wolfgang Bangerth
+ * This geometry model implements an (3d) ellipsoidal_chunk geometry which can be non-coordinate parallel and
+ * can have an initial topography on it.
+ * @author This plugin is a joined effort of Menno Fraters, D Sarah Stamps, Wolfgang Bangerth and Anne Glerum

```

Comment:
Doesn't this comment just reproduce what you already had in the .h file? If so, just remove it altogether.